lib/interface_quda.cpp

```diff
@@ -5835,7 +5822,7 @@ void contract(const cudaColorSpinorField x, const cudaColorSpinorField y, void *
   }
 }
 
-double qChargeCuda ()
+double qChargeQuda()
```

Comment:
I think you missed multigrid_evolve_test uses this function.

Reply:
Thanks for the catch.